2011.final-value-of-variable-after-performing-operations.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * @param {string[]} operations
+ * @return {number}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/final-value-of-variable-after-performing-operations/
+*/
+
+var finalValueAfterOperations = function(operations) {
+    let X = 0;
```

Comment:
Can you make it descriptive name?

Reply:
I modified the variable name by using "finalValueOfX" instead of "X"